purchase_order_variant_mgmt/models/purchase_order.py

```diff
@@ -17,8 +17,8 @@ class PurchaseOrderLine(models.Model):
     state_purchase_order_variant_mgmt = fields.Selection(
         related="order_id.state", readonly=True, string="Order status"
     )
-    product_attribute_value_ids = fields.Many2many(
-        comodel_name="product.attribute.value",
-        related="product_id.attribute_value_ids",
+    product_template_attribute_value_ids = fields.Many2many(
```

Comment:
@sonhd91 I'm wondering in which measure the field name shouldn't be kept like before as it will maybe add some work to people during migration if they inherited from this.

IMHO we can keep field name, changing just the comodel_name and related field attributes.